var/spack/repos/builtin/packages/py-isort/package.py

```diff
@@ -13,16 +13,33 @@ class PyIsort(PythonPackage):
     pypi = "isort/isort-4.2.15.tar.gz"
 
     version("5.12.0", sha256="8bef7dde241278824a6d83f44a544709b065191b95b6e50894bdc722fcba0504")
+    version("5.11.5", sha256="6be1f76a507cb2ecf16c7cf14a37e41609ca082330be4e3436a18ef74add55db")
     version("5.10.1", sha256="e8443a5e7a020e9d7f97f1d7d9cd17c88bcb3bc7e218bf9cf5095fe550be2951")
     version("5.9.3", sha256="9c2ea1e62d871267b78307fe511c0838ba0da28698c5732d54e2790bf3ba9899")
     version("5.9.1", sha256="83510593e07e433b77bd5bff0f6f607dbafa06d1a89022616f02d8b699cfcd56")
-    version("4.3.20", sha256="c40744b6bc5162bbb39c1257fe298b7a393861d50978b565f3ccd9cb9de0182a")
-    version("4.2.15", sha256="79f46172d3a4e2e53e7016e663cc7a8b538bec525c36675fcfd2767df30b3983")
+    version(
+        "4.3.20",
```

Comment:
Deprecating these means deprecating support for Python 3.6.0, correct?

Reply:
Well, I guess we've always only support isort 5+ anyway, so it doesn't really matter whether or not these are deprecated.

Reply:
Deprecation is just to make sure they are not used accidentally by `spack style`.